_includes/misc/hunspell-dictionaries-path.md

```diff
@@ -0,0 +1,19 @@
+The `ephox.spelling.hunspell-dictionaries-path` element is used to define the location of the Hunspell dictionaries. When the setting is not provided, Hunspell dictionaries are not supported.
+
+Requirements: 
+
+- The directory containing the Hunspell dictionaries must conform to the file structure defined in {{MISSING}} below
```

Comment:
`{{MISSING}}` ?

Reply:
Fixed in https://github.com/tinymce/tinymce-docs/pull/2048/commits/dc8efd92e844b24b175620d0e51176af9649bedb